tests/zfs-tests/tests/functional/write_dirs/setup.ksh

```diff
@@ -51,4 +51,7 @@ DISK=${DISKS%% *}
 
 log_must set_partition $SLICE "" $SIZE $DISK
 
+if is_linux; then
+	export SLICE=1
+fi
```

Comment:
That's subtle.  Does this now mean we can enable `write_dirs_002_pos` in the runfile?  If so you should update the runfile accordingly in this patch.